pkgs/tools/virtualization/ec2instanceconnectcli/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, boto3 }:
+
+buildPythonPackage rec {
+  pname = "ec2instanceconnectcli";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-VaCyCnEhSx1I3bNo57p0IXf92+tO1tT7KSUXzO1IyIU=";
+  };
+
+  propagatedBuildInputs = [ boto3 ];
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```

per recent contributing guide change

---

Thanks. I have updated accordingly and force pushed.